### PR TITLE
Home Work 10

### PR DESCRIPTION
**Pull request by TanyaB777**

### CODE REVIEW

File: src/main/java/hw_10/Person.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        this.age = age;
+    }
+
+    public double getSalary() {
+        return salary;
+    }
+
+    public void setSalary(double salary) {
+        this.salary = salary;
+    }
```
Comment (AntBelyi) [inline src/main/java/hw_10/Person.java]: please remove unused methods

File: src/main/java/hw_10/Task3.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        peopleList.add(person3);
+        peopleList.add(person4);
+        peopleList.add(person5);
+
+        System.out.println("List of persons:");
+        System.out.println(peopleList);
+
+        //for (Person person : peopleList) {
+        //    System.out.println(person);
+        //}
```
Comment (AntBelyi) [inline src/main/java/hw_10/Task3.java]: can be removed?

File: src/main/java/hw_10/Task3.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        List<String> peopleUnder70 = peopleList.stream()
+                .filter(person -> person.getAge() < 70)
+                .map(Person::getName)
+                .toList();
+
+        System.out.println(peopleUnder70);
+
+        //for (String name : peopleUnder70) {
+        //    System.out.println(name);
+        //}
```
Comment (AntBelyi) [inline src/main/java/hw_10/Task3.java]: can be removed?

File: src/main/java/hw_10/Person.java (hunk 1 above)
Comment (TanyaB777) [reply src/main/java/hw_10/Person.java]: fixed

File: src/main/java/hw_10/Task3.java (hunk 2 above)
Comment (TanyaB777) [reply src/main/java/hw_10/Task3.java]: fixed

File: src/main/java/hw_10/Task3.java (hunk 3 above)
Comment (TanyaB777) [reply src/main/java/hw_10/Task3.java]: fixed

**Pull request merged**